Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ruby:2.7.1-alpine
```

Comment:
I'll note to update to `2.7.2` later to keep up with the security updates. I'll do that after this has been done to avoid delays and blockers.